Release Notes for patch release 4.1.9 (#1755)
spectrocloud/librarium: docs/docs-content/release-notes.md

tags: ["release-notes"]
---

> ## November 7, 2023 - Release 4.1.9
> 
> This release contains the following minor bug fixes related to protected Helm Chart and Open Container Initiative (OCI) registries.
> 

Reviewer: ```suggestion
## November 7, 2023 - Release 4.1.9

This release contains the following minor bug fixes related to protected Helm Chart and Open Container Initiative (OCI) registries.

### Bug Fixes
```

We set the precedent to always include the header for the category. I checked because I was also unsure about this 😬  but found https://docs.spectrocloud.com/4.0.x/release-notes#august-31-2023---release-407

Author: @karl-cardenas-coding Oops, I forgot the heading.

> - Where Palette displayed the UID of private or protected Helm Chart registries in a drop-down menu during cluster profile creation, Palette now correctly displays the registry name.

Reviewer: ```suggestion
- An issue where Palette was displaying the UID of protected Helm Chart and Open Container Initiative (OCI) registries instead of the expected registry name is now corrected.
```
Rita, could we simplify the sentence to something along these lines maybe?

> - The issue is fixed where Palette did not allow charts from private or protected Helm Chart and OCI registries to be attached to cluster profiles. 

Reviewer: ```suggestion
- The issue is fixed where Palette did not allow Helm charts from private or protected Helm Chart and OCI registries to be attached to cluster profiles. 
```

> - The issue is resolved where Palette did not display packs for selection that belonged to a protected Helm Chart registry, preventing cluster profile creation. 

Reviewer: @ritawatson I'm a bit confused about this sentence. Is this sentence trying to convey that some packs were not available for selection? And as a result, the user could not create a cluster profile with those packs?

Author: Yes.
